packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -133,23 +139,38 @@ export class EditManager<
 		public readonly changeFamily: TChangeFamily,
 		public readonly localSessionId: SessionId,
 		private readonly mintRevisionTag: () => RevisionTag,
+		private readonly logger?: ITelemetryLoggerExt,
 	) {
 		this.trunkBase = {
 			revision: "root",
 			change: changeFamily.rebaser.compose([]),
 		};
 		this.sequenceMap.set(minimumPossibleSequenceId, this.trunkBase);
+
+		if (this.logger !== undefined) {
+			this.telemetryEventBatcher = new TelemetryEventBatcher(
+				{
+					eventName: "rebaseProcessing",
+					category: "performance",
+				},
+				this.logger,
+				1000, // TODO: Replace pattern.
```

Comment:
Not sure what "replace pattern" means, but do we have a number we want to go with for the batching right now? @yann-achard-MS you might be the one who know best if a hardcoded number here would be serviceable to start. How many rebase operations do we expect a given client to see normally, such that we get some messages out of this, but not too many? In other places we have something [like this](https://github.com/alexvy86/FluidFramework/blob/1c6c72689e27b8c140d6250814481d26ae097132/packages/dds/shared-object-base/src/sharedObject.ts#L162) that gives us flexibility to tweak the number (getting a mc/monitoringContext just requires a call [like this](https://github.com/alexvy86/FluidFramework/blob/1c6c72689e27b8c140d6250814481d26ae097132/packages/dds/shared-object-base/src/sharedObject.ts#L133) around the logger).

Reply:
The first link you attached is exactly what I meant by 😄. Working on it right now.